src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,14 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
let's use only `flatMap()` here
```suggestion
                .flatMap(n -> Arrays.stream(n.split(",")))
```

Reply:
done